src/assets/scss/components/PatientLookup.scss

```
@@ -3,22 +3,19 @@
   flex-direction: row;
   margin: 1rem;
   .InputRowField {
-    width: 25%;
+    width: 30em;
     padding-right: 1rem;
     display: flex;
     flex-direction: column;
     justify-content: flex-end;
     label {
       font-weight: bold;
     }
-    #toggle-Privacy-Level {
-      width: 1000px;
-    }
   }
 }
 
 .PatientLookupWrapper {
-  padding-bottom: 0px;
+  padding-bottom: 0em;
```

```suggestion
  padding-bottom: 0;
```
Units not needed when value is 0